### PR TITLE
Version 0.11.0!

### PR DESCRIPTION
**Pull request by beorn7**
@juliusv

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,27 @@
...
+* [FEATURE] Add offset operator.
+* [FEATURE] Add floor, ceil and round functions.
+* [CHANGE] Change instance identifiers to be host:port.
+* [CHANGE] Dependency management and vendoring changed/improved.
+* [CHANGE] Flag name changes to create consistency between various Prometheus
+  binaries.
+* [CHANGE] Show unlimited number of metrics in autocomplete.
+* [CHANGE] Add query timeout.
+* [CHANGE] Remove labels on persist error counter.
+* [ENHANCEMENT] Various performance improvement for sample ingestion.
```
Comment (juliusv) [inline CHANGELOG.md]: s/improvement/improvements/

**Comment by juliusv** [comment]
Woot :+1: (besides comment)

**Comment by matthiasr** [comment]
commit says "0.10.0"

**Comment by beorn7** [comment]
All fixed. Thanks.

**Pull request merged**